src/cf-buttons/src/atoms/buttons.less

```diff
@@ -72,13 +74,14 @@
   }
 
   &:hover,
-  &.hover {
+  &.hover,
+  &:focus,
```

Comment:
I know it's minor, but it reduces another rule from the final file. I've started trying to make gains everywhere I can think of and my hope is if we make enough small gains, we can make one large impact. I wish Less could combine these, but it's not that intuitive.

__Before__
![screen shot 2016-09-22 at 3 38 47 pm](https://cloud.githubusercontent.com/assets/1280430/18765309/b99b844c-80da-11e6-8515-c4ab2400509f.png)